singlestoredb/tests/test_fusion.py

```diff
@@ -463,3 +463,252 @@ def test_create_drop_workspace_group(self):
                 mgr.workspace_groups[wg_name].terminate(force=True)
             except Exception:
                 pass
+
+
+@pytest.mark.management
+class TestJobsFusion(unittest.TestCase):
+
+    id: str = secrets.token_hex(8)
+    notebook_name: str = 'Scheduling Test.ipynb'
+    dbname: str = ''
+    dbexisted: bool = False
+    manager: None
+    workspace_group: None
+    workspace: None
+    job_ids = []
+
+    @classmethod
+    def setUpClass(cls):
+        sql_file = os.path.join(os.path.dirname(__file__), 'test.sql')
```

Comment:
If you aren't accessing any of the test data in your tests, you could remove this.

Reply:
There wouldn't be a database if you removed this. It's pretty small, so it doesn't really hurt if you want to leave it in.